lib/MentionsForm/styles.scss

```diff
@@ -164,6 +164,10 @@ $mention-suggestions-max-height: 500px;
 
 }
 
+.mention-form__input__suggestions {
+  z-index: 2 !important;
```

Comment:
❗️ important

Reply:
Gross I know, the mentions package adds in some inline styles, so its the only way to override 🤢